lib/sensu-plugins-cpu-checks/version.rb

```diff
@@ -2,7 +2,7 @@ module SensuPluginsCpuChecks
   module Version
     MAJOR = 1
     MINOR = 0
-    PATCH = 0
+    PATCH = 1
```

Comment:
can you remove this and I will bump the version after the merge, there are likely several other unreleased fixes.